docs/sources/http_api/admin.md

```diff
@@ -612,9 +612,13 @@ Only works with Basic Authentication (username and password). See [introduction]
 
 See note in the [introduction]({{< ref "#admin-api" >}}) for an explanation.
 
-| Action              | Scope                  | Provision entity |
-| ------------------- | ---------------------- | ---------------- |
-| provisioning:reload | services:accesscontrol | accesscontrol    |
+| Actions              | Scopes                      | Provision entities |
```

Comment:
Should be singular - Action, Scope, and Provision entity.

Reply:
Thank you for the pushback, @vtorosyan. I brought this to the Docs Squad last week to come to a collective decision. The result of which is that we can consistently advise folks on using singular as opposed to plural, by default.